posthog/batch_exports/sql.py

```diff
@@ -289,6 +289,37 @@
 )
 """
 
+CREATE_EVENTS_BATCH_EXPORT_VIEW_RECENT_DISTRIBUTED = f"""
```

Comment:
Like we discussed, this could be a good chance to start not doing these views anymore.

Reply:
ok, will try to see if I can remove it